device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/port_config.ini

```diff
@@ -1,33 +0,0 @@
-# name          lanes             alias         index   speed
```

Comment:
I believe this file may still be expected by some containers in Sonic still.

Reply:
Reverted